blog/2024-01-25-website_update.md

```diff
@@ -0,0 +1,33 @@
+---
+slug: website_update
+title: Website updated
+authors: [alo.alt]
+tags: [wayang]
+---
+
+We're updated our website and use now Docusaurus.
+
+<!--truncate-->
+# Website updated
+
+Author: [2pk03](https://github.com/2pk03)
+
+We switched to a new CMS. That's all.
+
+## Cheatsheet
```

Comment:
This is an example of the Docusaurus. I think it should be deleted

Reply:
Na, I did that to have a blog entry :)